docs/languages/en/modules/zend.input-filter.intro.rst

```diff
@@ -167,6 +167,197 @@ appropriate object. You may create either ``Input`` or ``InputFilter`` objects i
+            }
+
+            /**
+             * Loads the validators
+             */
+            protected function setValidators()
+            {
+                $this->setNameValidator();
+            }
+
+            /**
+             * Creates a validator to check the name property
+             */
+            protected function setNameValidator()
+            {
+                $this->add(
+                    $this->inputFactory->createInput(
+                        array(
+                            'name'       => 'name',
+                            'required'   => true,
+                            'validators' => array(
+                                array(
+                                    'name' => 'not_empty',
+                                ),
+                                array(
+                                    'name' => 'string_length',
+                                    'options' => array(
+                                        'min' => 8
+                                    )
+                                ),
+                            )
+                        )
+                    )
+                );
+            }
+        }
+
+        /**
+         * Filter to ensure an email property is set and > 8 characters and is valid
+         */
+        class EmailInputFilter extends InputFilter
+        {
+            /** @var InputFactory */
+            protected $inputFactory;
+
+            public function __construct()
+            {
+                $this->inputFactory = new InputFactory();
+                $this->setValidators();
+            }
+
+            /**
+             * Loads the validators
+             */
+            protected function setValidators()
+            {
+                $this->setEmailValidator();
+            }
+
+            /**
+             * Creates a validator to check the name property
+             */
+            protected function setEmailValidator()
+            {
+                $this->add(
+                    $this->inputFactory->createInput(
+                        array(
+                            'name'       => 'email',
+                            'required'   => true,
+                            'validators' => array(
+                                array(
+                                    'name' => 'not_empty',
+                                ),
+                                array(
+                                    'name'    => 'string_length',
+                                    'options' => array(
+                                    'min'     => 8
+                                ),
+                                array(
+                                    'name'    => 'email_address',
+                                )
+                            )
+                        )
+                    )
+                );
+            }
+        }
+
+        class SimplePerson implements InputFilterAwareInterface
```

Comment:
Please don't use the `InputFilterAwareInterface` in examples: it's a bad design.